src/Go.Windows/Logger.cs

```diff
@@ -0,0 +1,107 @@
+﻿namespace Go.Windows
+{
+    using System;
+    using System.ComponentModel.Composition;
+    using System.Text;
+    using Microsoft.VisualStudio;
+    using Microsoft.VisualStudio.Shell;
+    using Microsoft.VisualStudio.Shell.Interop;
+
+    [Export]
```

Comment:
Let’s export your logger interface here. `[Export(typeof(Logger))]`

That way it can be imported by Go.Editor